### DISCUSSION TITLE
In what country would you find The 🔥⛰️?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#812] [Galaxy Trivia] [answered]
**Bonus:** How/When did it all start?

**Comment by ParanoidUser** [comment]
@vituhonorato / @promisedEgnr, any ideas? 😶‍🌫️

**Reply by SANKARAMDAS** [reply, marked as answer]
Yanar Dagh (Azerbaijani: Yanar Dağ, meaning "burning mountain") is a natural gas fire which blazes continuously on a hillside on the Absheron Peninsula on the Caspian Sea near Baku, the capital of Azerbaijan (a country which itself is known as "the Land of Fire").

Unlike mud volcanoes, the Yanar Dagh flame burns fairly steadily, as it involves a steady seep of gas from the subsurface. It is claimed that the Yanar Dagh flame was only noted when accidentally lit by a shepherd in the 1950s. There is no seepage of mud or liquid, which distinguishes it from the nearby mud volcanoes of Lokbatan or Gobustan.

https://en.wikipedia.org/wiki/Yanar_Dag#:~:text=Yanar%20Dagh%20(Azerbaijani%3A%20Yanar%20Da%C4%9F,the%20Land%20of%20Fire%22).

**Comment by peterbaccaro** [comment]
The "Burning Mountain" is located in **Australia**. It is also known as Mount Wingen or Burning Mountain Nature Reserve. The Burning Mountain is a natural phenomenon where an underground coal seam has been burning for thousands of years, resulting in smoke and occasional flames emanating from the ground. It is situated near Wingen in New South Wales, Australia. This unique geological feature is a popular tourist attraction and is considered the oldest known coal fire in the world.

The fire reportedly ignited **6,000 years ago** and surprisingly, some scientists estimate it to be even older.

Their origin stories talk about **a widow whose tears ignited the fire**, or **the torch of a warrior captured by the 'Evil One' under the mountain**. Natural causes are the most likely source.

https://en.wikipedia.org/wiki/Burning_Mountain
https://www.sciencetimes.com/articles/35341/20220103/burning-mountain-sacred-site-underground-fire-6-000-years-beyond.htm

![image](https://github.com/ParanoidUser/yolo/assets/74666079/bd1059bc-1ced-47a9-8d41-5d8a9b27da41)
